engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -166,18 +172,18 @@ class EngineJobExecutionActor(replyTo: ActorRef,
 
     case Event(HashError(t), data: SucceededResponseData) =>
       disableCallCaching(t)
-      saveJobCompletionToJobStore(data)
+      saveJobCompletionToJobStore(data.copy(hashes = Some(Failure(t))))
     case Event(HashError(t), data: ResponsePendingData) =>
       disableCallCaching(t)
-      stay
+      stay using data.copy(hashes = Some(Failure(t)))
```

Comment:
I agree with this case. A lot of the other `Some`s are in `should be(Some(_))`  and I intend to keep those as-is